test/performance/odp_ipsecfwd.c

```diff
@@ -1213,6 +1214,9 @@ static void teardown_test(const prog_config_t *config)
 
 	if (config->compl_q != ODP_QUEUE_INVALID)
 		(void)odp_queue_destroy(config->compl_q);
+
+	free(config->sa_conf_file);
+	free(config->fwd_conf_file);
```

Comment:
Execution can end up here without ever going through init_config(), so the frees would be called with uninitialized pointers.

Reply:
`init_config()` was called too early also before (`odp_schedule_config()` could have failed), done earlier in v2.